backend/EasyMeets.Core/EasyMeets.Core.DAL/Entities/TeamMember.cs

```diff
@@ -0,0 +1,15 @@
+using EasyMeets.Core.DAL.Entities.Enums;
+
+namespace EasyMeets.Core.DAL.Entities;
+
+public class TeamMember
```

Comment:
Why you didn't use Entity here?

Reply:
we have composite primary key for this table and we don't need Entity here. Or it would be better to use Entity and make Id-field primary key ?

Reply:
I believe it would be better to have Id here as a combined primary key can cause some issues in the future